website/docusaurus.config.js

```diff
@@ -95,11 +95,11 @@ module.exports = {
         },
         {
           to: 'docs/introduction',
-          activeBaseRoute: 'docs',
+          activeBasePath: 'docs',
           label: 'Docs',
           position: 'left',
         },
-        {to: 'blog', label: 'Blog', position: 'left'},
+        {to: 'blog', activeBasePath: 'blog', label: 'Blog', position: 'left'},
```

Comment:
>  activeBasePath: 'blog'

It’s not necessary here, the Blog link is highlighted perfectly without it, isn’t it?

Reply:
For now, nope. But we never know if that might change in future. Anyway no harm adding IMO. I'm also ok with removing but I think it's good for education.

Reply:
This seems to me a "bad example" in this case, we should show that the `activeBasePath` option is more of a workaround that should be used wisely.

Reply:
Sure, will remove.